mos_tests/functions/common.py

```diff
@@ -2,58 +2,57 @@
 from time import sleep, time
 import urllib2
 import yaml
+import platform
 
 
-def check_stack(stack_name, heat):
+def is_stack_exists(stack_name, heat):
     """ Check the presence of stack_name in stacks list
-        :param heat: Heat API client connection point
         :param stack_name: Name of stack
+        :param heat: Heat API client connection point
         :return True or False
     """
-    if stack_name in [s.stack_name for s in heat.stacks.list()]:
-        return True
-    return False
+    return stack_name in [s.stack_name for s in heat.stacks.list()]
 
 
-def get_stack_id(heatclient, stack_name):
+def get_stack_id(heat_client, stack_name):
     """ Check stack status
-        :param heatclient: Heat API client connection point
+        :param heat_client: Heat API client connection point
         :param stack_name: Name of stack
         :return Stack uid
     """
-    if check_stack(stack_name, heatclient):
-        stack_dict = {s.stack_name: s.id for s in heatclient.stacks.list()}
+    if is_stack_exists(stack_name, heat_client):
```

Comment:
I'll include it in next commit, since it is a common part and it were not implemented by me, initially.